tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx

```diff
@@ -71,8 +81,33 @@ const SubmissionHistory = ({ filterValues }) => {
     }
   }, [hasFetchedFiles, files, dispatch, filterValues])
 
+  let submissionThreshold = new Date('05-31-2024')
+  const fileIsOutdated = (f) => {
+    let created_date = new Date(f.createdAt)
+    return created_date < submissionThreshold
+  }
+
+  const hasOutdatedSubmissions = () =>
+    files.some((e, i, a) => fileIsOutdated(e))
```

Comment:
Could we give the params of the functor a bit more descriptive names?

Reply:
javascript bad habits 😄